hedera-node/hedera-config/src/testFixtures/java/com/hedera/node/config/testfixtures/HederaTestConfigBuilder.java

```diff
@@ -175,6 +175,7 @@ public static TestConfigBuilder create() {
                 .withConfigDataType(StakingConfig.class)
                 .withConfigDataType(StatsConfig.class)
                 .withConfigDataType(TokensConfig.class)
+                .withConverter(new ScaleFactorConverter())
```

Comment:
Is this change necessary, or can we revert it?

Reply:
Reverted, I guess something happened when merging with master